test/TestUtilities/Test.Utility/SimpleTestSetup/SimpleTestProjectContext.cs

```diff
@@ -334,7 +337,14 @@ public void AddProjectToAllFrameworks(params SimpleTestProjectContext[] projects
 
         public void Save()
         {
-            Save(ProjectPath);
+            string path = ProjectPath;
+
+            if (!Path.IsPathRooted(ProjectPath))
+            {
+                path = Path.Combine(SolutionRoot ?? Path.GetDirectoryName(ProjectPath), ProjectPath);
+            }
```

Comment:
Without much context, I ask this question: This looks a lot like the logic in MSBuildAPIUtility and because this is in a test object, it makes me wonder if we're relying on this test code matching the product code rather than exercising the product code directly?